src/assets/styles/base.sass

```diff
@@ -76,6 +76,9 @@ abbr[title]
 #sidebarSearchForm
   font-size: 14px
   margin: 28px 0 28px auto
+  // match Inkeep's dimensions to avoid visible reflow
+  width: 262.5px
+  height: 40px
 
   input
```

Comment:
This `input` element doesn't seem to exist anymore, with the Inkeep based search. The specific style for the `button` element may also be unnecessary now.

Shall we remove it or leave it present, in case we switch back to another search form (with a real input field)?